package.json

```diff
@@ -209,6 +209,7 @@
     "**/@babel/preset-react": "^7.16.7",
     "**/@babel/preset-typescript": "^7.16.7",
     "**/@babel/runtime": "^7.17.2",
+    "**/@definitelytyped/typescript-versions": "^0.0.121",
```

Comment:
Can you share more info on why you landed on this version and why this fixes the issue?

Reply:
`dtslint` specifies its dependency on `@definitelytyped/typescript-versions` as `latest`, which leads to this problem. Having the version resolved this way doesn't really change the version used but eliminates the warning.

`0.0.121` was the latest version of this package when I authored this PR (it changed since - I'll update it in a bit).

I've seen this problem in other projects as well: https://github.com/thefrontside/bigtest/pull/469